h/templates/activity/search.html.jinja2

```diff
@@ -95,6 +95,82 @@
         </li>
       {% endfor %}
     </ol>
+
+    {% if group %}
+    <aside class="search-result-sidebar">
+      <h1 class="search-result-sidebar__title">{{ group.name }}</h1>
+
+      {% if group.description %}
+      <section class="search-result-sidebar__section">
+        {% for paragraph in group.description.split('\n') %}
+          <p>{{ paragraph }}</p>
+        {% endfor %}
+      </section>
+      {% endif %}
+
+      <section class="search-result-sidebar__section">
+        <dl>
+          <dt class="search-result-sidebar__dt">Annotations:</dt>
+          <dd class="search-result-sidebar__dd">{{ total }}</dd>
+
+          <dt class="search-result-sidebar__dt">Created:</dt>
+          <dd class="search-result-sidebar__dd">{{ group.created }}</dd>
+        </dl>
+        <div style="clear: both;"></div>
+      </section>
+
+      <section class="search-result-sidebar__section">
+        <ul>
+          {% if group_edit_url %}
+            <li>
+              <a class="search-result-sidebar__link"
+                 href="{{ group_edit_url }}">
+                {% trans %}Edit group{% endtrans %}
+              </a>
+            </li>
+          {% endif %}
+          {% if show_group_leave_button %}
+            <li>
+              {#- This form element is needed for Internet Explorer because it
+                  doesn't support the `form` attribute. See commit message. #}
```

Comment:
These comments get removed by Jinja2 in the HTML that the server renders the template to